### PR TITLE
helm charts - extract ingress class to global variable

### PR DESCRIPTION
**Pull request by sparkoo**
Signed-off-by: Michal Vala <mvala@redhat.com>

### What does this PR do?
Extracts `ingressClass` to variable in helm chart.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13927

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
@sparkoo: FYI: [E2E Happy path tests failed](https://github.com/eclipse/che/pull/14201#issuecomment-520469079) because there was flaky unit test which doesn't related to your changes. So, I am restarting tests again.

**Comment by dmytro-ndp** [comment]
crw-ci-test

**Comment by dmytro-ndp** [comment]
crw-ci-test

File: deploy/kubernetes/helm/che/values.yaml [hunk 1]
```diff
@@ -34,6 +34,7 @@ global:
   ingressDomain: 192.168.99.100.nip.io
   # See --annotations-prefix flag (https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/cli-arguments.md)
   ingressAnnotationsPrefix: "nginx."
+  defaultIngressClass: "nginx"
```
Comment (sleshchenko) [inline deploy/kubernetes/helm/che/values.yaml]: Please consider renaming it to `ingressClass` to be consistent with `ingressAnnotationsPrefix`.

Comment (sparkoo) [reply deploy/kubernetes/helm/che/values.yaml]: `defaultIngressClass` is from Operator implementation https://github.com/eclipse/che-operator/blob/fbeb80885d76f1dc1a662967f7a7277de7f9738c/pkg/deploy/defaults.go#L32

Comment (sleshchenko) [reply deploy/kubernetes/helm/che/values.yaml]: Note that you referenced name variable in go code.
and when a user wants to specify ingress class, he defines `cr.Spec.K8SOnly.IngressClass` See https://github.com/eclipse/che-operator/blob/8837474a0fe4082679af4b2393ca55992cc5f75c/pkg/deploy/ingress.go#L29

In helm case, a user would define `defaultIngressClass`.
I still think that it makes sense to rename it to `IngressClass`

Comment (sparkoo) [reply deploy/kubernetes/helm/che/values.yaml]: yes, make sense. fixed.

**Review by sleshchenko — APPROVED** [review]

**Comment by sparkoo** [comment]
won't fix in favor of #14230

**Pull request closed without merging**